packages/user/src/assets/css/login.css

```diff
@@ -2,6 +2,12 @@
   justify-items: center;
 }
 
+.login > .content > .reminder {
+  position: relative;
+  left: 15rem;
+  bottom: 2rem;
+}
+
```

Comment:
I am not sure if we really need this. Also 15rem left is a large space to move, it can be looking bad in mobile ui please check.